lib/ansible/modules/cloud/azure/azure_rm_resource.py

```diff
@@ -109,12 +109,12 @@
 EXAMPLES = '''
   - name: Update scaleset info using azure_rm_resource
     azure_rm_resource:
-      resource_group: "{{ resource_group }}"
+      resource_group: myResourceGroup
       provider: compute
       resource_type: virtualmachinescalesets
-      resource_name: "{{ scaleset_name }}"
+      resource_name: myVmss
```

Comment:
Here you are using 'myVmss' but in other places you have `myVmssName` - I'd prefer `myVmss` to match `myResourceGroup`.

Reply:
yeah, fixed.
I will have more prs following, regarding naming conventions.
just subscription ids and resource groups were most obvious

Reply:
great - start with the low-hanging fruit for sure